ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -98,9 +105,11 @@ def __init__(self, instance):
             "connection_name:{}".format(self.connection_name),
         ]  # type: List[str]
         tags.extend(custom_tags)
-        if host or port:
+        if host and not override_hostname:
             # 'host' is reserved and 'mq_host' is used instead
-            tags.extend({"mq_host:{}".format(host), "port:{}".format(port)})
+            tags.append("mq_host:{}".format(host))
+        if host or port:
```

Comment:
```suggestion
        if port:
```
nit

Reply:
This will change behavior: https://github.com/DataDog/integrations-core/pull/11223#discussion_r793722568